src/mon/MonCommands.h

```diff
@@ -869,6 +869,31 @@ COMMAND("osd pool get-quota " \
         "name=pool,type=CephPoolname ",
         "obtain object or byte limits for pool",
         "osd", "r", "cli,rest")
+COMMAND("osd pool application enable " \
```

Comment:
Yeah, hopefully, but that's obviously not the case right now so we shoulda void degrading our UI as much as possible as we add these new constraints for them to deal with?